benchmarks/work_precision_sets/time_vs_abstols.py

```diff
@@ -0,0 +1,126 @@
+import pybamm
```

Comment:
Will make some general comments on this script on how to simplify the code, but they apply to all the scripts I believe.